### PR TITLE
Fix gh-1824: Add note about real names

### LINKED ISSUE #1824: Add "Real Name" reminder to Teacher Account flow
Currently in the account creation flow for Teacher Accounts we do not tell Teachers to choose a username that does not include personally identifiable information. This is what the flow currently looks like: 

image

We would like to add the text "**Please do not use any portion of your real name in your username.**" at the top of the page, written in bold. Here is a quick mock-up of what we are thinking:

image

### PR DESCRIPTION
**Pull request by JoelGritter**
Fixes gh-1824

The edit made should be visible on teacher registration.

### Resolves:

Fixes #1824 by appending sentence to existing description on the navigation card.

### Changes:

"Please do not use any portion of your real name in your username." is now added to the 'description' for the username step of registration. The added text should be in bold. Thus the text should now read:
> Fill in the following forms to request an account. The approval process may take up to one day. **Please do not use any portion of your real name in your username.**

### Testing

- [ ] Teacher registration should have the note about real names in bold in the registration

### CODE REVIEW

File: src/l10n.json [hunk 1]
```diff
@@ -133,7 +133,7 @@
     "registration.selectCountry": "select country",
     "registration.studentPersonalStepDescription": "This information will not appear on the Scratch website.",
     "registration.showPassword": "Show password",
-    "registration.usernameStepDescription": "Fill in the following forms to request an account. The approval process may take up to one day.",
+    "registration.usernameStepDescription": "Fill in the following forms to request an account. The approval process may take up to one day. <b>Please do not use any portion of your real name in your username.</b>",
```
Comment (chrisgarrity) [inline src/l10n.json]: This message cannot have HTML tags in it.

**Comment by JoelGritter** [comment]
@chrisgarrity So basically add another string to the .json file and then include the FormattedMessage for the new string in the `<p>` on the form? (And put `<b>` tags around the FormattedMessage)

**Comment by chrisgarrity** [comment]
@JoelGritter correct, add the text as a new string, and wrap the `FormattedMessage`  with the `<b>` tag in the form. That's probably better for translators in any case. As a separate string all the current translations will be valid, and translators just have to translate the new text. If they're combined then all previous translations are invalidated, and translators have to redo them.

**Comment by JoelGritter** [comment]
@chrisgarrity Okay, no problem. I'll push a new change to this PR when I've got some free time. (Probably in a couple hours)

**Comment by JoelGritter** [comment]
@chrisgarrity Just completed the changes you requested.

**Comment by JoelGritter** [comment]
Looks like I'll have to fix some issues, sorry

File: src/components/registration/steps.jsx [hunk 2]
```diff
@@ -182,6 +182,9 @@ class UsernameStep extends React.Component {
                         this.props.description
                     ) : (
                         <intl.FormattedMessage id="registration.usernameStepDescription" />
+                        <b>
```
Comment (chrisgarrity) [inline src/components/registration/steps.jsx]: I think it'll be fine if you just put a `<span>...</span>` around the whole message.

Comment (JoelGritter) [reply src/components/registration/steps.jsx]: Sure, I can try doing that!

**Comment by JoelGritter** [comment]
@chrisgarrity I've made the suggested change. Once this is resolved I could close this PR and make a new one so that there's only one commit to merge.

As a side note, is there a way to test with Travis on my local instance?

**Comment by chrisgarrity** [comment]
@JoelGritter don't worry about closing the PR - I can squash all the commits into one when I merge. 

On your local instance you should be able to run `npm run test` to run all the tests that travis runs. To just check for lint errors you can also run `make lint`.

File: src/components/registration/steps.jsx [hunk 3]
```diff
@@ -768,11 +773,11 @@ class OrganizationStep extends React.Component {
                 id: `teacherRegistration.${choice}`
             })
         }));
-        
+
```
Comment (chrisgarrity) [inline src/components/registration/steps.jsx]: Remove the unnecessary blank lines added.

Comment (JoelGritter) [reply src/components/registration/steps.jsx]: I'm going to blame Atom for this one. 😝

**Comment by JoelGritter** [comment]
@chrisgarrity Done! Apparently Atom has a default package called 'Whitespace' that automagically adds a single blank line at the end of the file.

**Comment by JoelGritter** [comment]
I'm going to make commits locally and test locally until I appease Travis.

File: src/components/registration/steps.jsx [hunk 4]
```diff
@@ -1593,4 +1598,4 @@ module.exports.TeacherApprovalStep = IntlTeacherApprovalStep;
 module.exports.ClassInviteNewStudentStep = IntlClassInviteNewStudentStep;
 module.exports.ClassInviteExistingStudentStep = IntlClassInviteExistingStudentStep;
 module.exports.ClassWelcomeStep = IntlClassWelcomeStep;
-module.exports.RegistrationError = IntlRegistrationError;
+module.exports.RegistrationError = IntlRegistrationError;
```
Comment (chrisgarrity) [inline src/components/registration/steps.jsx]: Our style guidelines do require the blank line at the end of the file, but there were extras in the middle of the file (lines 776 and 780), those may have been added by a code formatting add on to Atom :)

**Comment by JoelGritter** [comment]
@chrisgarrity Bingo, I got the tests to pass locally and on Travis!

**Pull request merged**